packages/material-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -53,20 +53,16 @@ const SelectWidget = ({
   onFocus,
   rawErrors = [],
 }: WidgetProps) => {
-  const { TextField, MenuItem } = useContext(MuiComponentContext);
+  const { TextField, MenuItem } = useMuiComponent();
   const { enumOptions, enumDisabled } = options;
 
   const emptyValue = multiple ? [] : '';
 
-  const _onChange = ({
-    target: { value },
-  }: React.ChangeEvent<{ name?: string; value: unknown }>) =>
+  const _onChange = ({ target: { value } }: React.ChangeEvent<{ name?: string; value: unknown }>) =>
```

Comment:
Is it possible to revert the whitespace changes? will make git blames easier in the future.